// motion/dist/main.cjs.js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var dom = require('@motionone/dom');
var animate = require('./animate.cjs.js');
var types = require('@motionone/types');



Object.defineProperty(exports, 'glide', {
	enumerable: true,
	get: function () { return dom.glide; }
});
Object.defineProperty(exports, 'inView', {
	enumerable: true,
	get: function () { return dom.inView; }
});
Object.defineProperty(exports, 'spring', {
	enumerable: true,
	get: function () { return dom.spring; }
});
Object.defineProperty(exports, 'stagger', {
	enumerable: true,
	get: function () { return dom.stagger; }
});
Object.defineProperty(exports, 'style', {
	enumerable: true,
	get: function () { return dom.style; }
});
Object.defineProperty(exports, 'timeline', {
	enumerable: true,
	get: function () { return dom.timeline; }
});
exports.animate = animate.animate;
Object.keys(types).forEach(function (k) {
	if (k !== 'default' && !exports.hasOwnProperty(k)) Object.defineProperty(exports, k, {
		enumerable: true,
		get: function () { return types[k]; }
	});
});
